### DISCUSSION TITLE
Is the Discord discontinued?

### DISCUSSION THREAD

**Discussion by Nantris** [software-mansion/react-native-gesture-handler#3160] [Q&A] [answered]
No channels are usable on it.

**Comment by Nantris** [comment]
Friendly bump. There's a major problem #3196 that makes `react-native-gesture-handler` really jank up our whole project - which is a big problem since so much of the ecosystem relies on it. I'd love to have some way to try to talk to people who might have had similar experiences, but the Discord is just two locked channels.

**Comment by m-bert** [comment, marked as answer]
HI @Nantris! What's your discord username? When you join our server, you should visit **_#welcome_** channel, read rules and then give yourself permissions to channels by adding roles. This should be enough to get access to specific channels.

**Reply by Nantris** [reply]
Thanks for your reply @m-bert. I see my misunderstanding. I'd recommend bolding and clarifying the text:

> To get access for desired sections, please choose appropriate roles from the following list.

I didn't understand that selecting emojis could modify permissions in a Discord server.

Thank you again.
